src/components/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
```

Comment:
- [ ] As you are using ES6 for your other components, I recommend you change the syntax of the App component to ES6 too. This will make your codebase look more organized and more professional.